test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -189,7 +189,7 @@ func TestEtcdStoragePath(t *testing.T) {
 					previousNonAlphaVersions.Insert(version)
 				}
 			}
-			if len(previousNonAlphaVersions) > 0 && !previousNonAlphaVersions.Has(expectedGVK.Version) {
+			if len(previousNonAlphaVersions) > 0 && !strings.Contains(expectedGVK.Version, "alpha") && !previousNonAlphaVersions.Has(expectedGVK.Version) {
```

Comment:
Fixes:
```sh
TestEtcdStoragePath/admissionregistration.k8s.io/v1beta1,_Resource=validatingadmissionpolicies
    etcd_storage_path_test.go:193: Previous releases understand non-alpha versions ["v1beta1"], but do not understand the expected current storage version "v1alpha1". This means a current server will store data in etcd that is not understood by a previous version.
```

Reply:
opened https://github.com/kubernetes/kubernetes/pull/120018 to switch the storage version and improve the etcd test to catch this

Reply:
This was an oversight. Thanks for opening the PR to fix.